src/utils/helpers/date/date.js

```diff
@@ -45,8 +49,21 @@ const DateHelper = {
    * @param {Object} options Override Moment JS options
    * @return {String} formatted date
    */
-  formatValue: (value, formatTo = isoDateFormat, options = {}) => {
```

Comment:
Please document this as a breaking change.

Reply:
added, thanks